inception-search-mtas/src/test/java/de/tudarmstadt/ukp/inception/search/index/mtas/MtasDocumentIndexTest.java

```diff
@@ -153,16 +154,16 @@ private void createProject(Project aProject) throws Exception
     }
 
     @SafeVarargs
-    private final void uploadDocument(Pair<SourceDocument, String>... aDocuments)
-        throws Exception
-    {
+    private final void uploadDocument(Pair<SourceDocument, String>... aDocuments) throws IOException {
         Project project = null;
         for (Pair<SourceDocument, String> doc : aDocuments) {
             project = doc.getLeft().getProject();
             
             try (InputStream fileStream = new ByteArrayInputStream(
                     doc.getRight().getBytes(UTF_8))) {
                 documentService.uploadSourceDocument(fileStream, doc.getLeft());
+            } catch (UIMAException e) {
+                e.printStackTrace();
```

Comment:
Printing a stack trace is not a good way to handle an exception. Rather throw it.